### ISSUE TITLE
File name field shouldn't allow slash (/) characters

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#1363]
In some cases (e.g., SaveAs) the file dialog will show `./project1/file1.css`. If you actually click OK, the dialog will actually append the string to the current path. For example, if you are in `folder1` and click Save, the file will be saved as `project1/folder1/project1/file1.css`.

We should never show or allow any sort of path within the file name field. It should only show or allow the last topic in a path (i.e., the fileaame part such as `file1.html`). One possibility is to put a regex on the textbox to turn on validation, where the regex allows any characters except "/". (And probably shouldn't allow leading or trailing spaces, or maybe strip them out at the end)

**Comment by childsb** [comment]
moving to m5.

**Comment by jhpedemonte** [comment]
-> M6

**Comment by JonFerraiolo** [comment]
-> @doronrosenberg maybe some magic with ValidationTextBox? Nice to have for M7, only if easy and safe.

**Comment by doronrosenberg** [comment]
Wouldn't it make more sense to escape / in this case?

**Comment by JonFerraiolo** [comment]
Nah. Users shouldn't be allowed to create filenames with "/" (or backslash for that matter) in their file names. It will mess up downstream logic where code (ours, dojo's, operating systems) search for "/" in fullpath strings and then split on that character.
